tutorials/feature_shennong/metadata.yml

```diff
@@ -0,0 +1,4 @@
+dimension: 41
```

Comment:
Can this be moved to [pyannote-shennong-features](https://github.com/bootphon/pyannote-shennong-features) repo?

Reply:
Deleted